src/cards/resurrect.ts

```diff
@@ -17,9 +17,9 @@ const spell: Spell = {
     replaces: [resurrect_weak_id],
     category: CardCategory.Soul,
     sfx: 'resurrect',
-    manaCost: 160,
+    manaCost: 120,
```

Comment:
reasoning?

Reply:
The real resurrect feels like a worse version of the other spells, since you can heal or purify or whatever else.

This change doesn't fix that (it shouldn't, players should be rewarded for finding those combos) but it does at least make the full resurrect feel competitive, which it should, given it's supposed to be an upgrade. Right now you kind've have to use one of the other combos instead